### PR TITLE
Fix broken link in USAGE.md

### PR DESCRIPTION
**Pull request by timolehto**
## What are you trying to accomplish?
Fix a broken link in docs.

## What approach did you choose and why?
Fixed it.

## What should reviewers focus on?
Verify that link now works.

## Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Code refactor (non-breaking change that doesn't add functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist

- [x] It is safe to simply rollback this change.

### CODE REVIEW

**Review by wildmaples — APPROVED** [review]

**Review by exterm — APPROVED** [review]
Thank you for your contribution! ❤️

**Comment by wildmaples** [comment]
Hi @timolehto, would you mind trying rebasing and trying to push to see if that triggers the CI tests?

File: USAGE.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ Here is a list of files generated:
 
 | File                        | Location     | Description |
 |-----------------------------|--------------|------------|
-| Packwerk configuration      | packwerk.yml | See [Setting up configuration file](#Setting-up-configuration-file) |
+| Packwerk configuration      | packwerk.yml | See [Setting up configuration file](#Setting-up-the-configuration-file) |
```
Comment (bogdanvlviv) [inline USAGE.md:50]: Let's also rephrase this to "Setting up the configuration file"? In other words there is missing "the"

**Comment by timolehto** [comment]
Rebased and rephrased. And seems like CI is now happy too.

**Review by tomstuart — APPROVED** [review]
All good. Thank you! 🎉
